src/biokbase/narrative/tests/test_app_util.py

```diff
@@ -331,37 +334,175 @@ def test_transform_param_value_textsubdata(value, expected):
     assert transform_param_value(None, value, spec) == expected
 
 
-mock_obj_ref = "wjriehl:1490995018528/Sbicolor2"
-ref_cases = [
-    (None, None),
-    ("foo/bar", "foo/bar"),
-    ("1/2/3", mock_obj_ref),
-    ("Sbicolor2", mock_obj_ref),
+tf_types = [None, "ref", "resolved-ref", "unresolved-ref", "putative-ref", "upa"]
+all_types = []
+for t in tf_types:
+    all_types.append(t)
+    if t is not None:
+        all_types.append(f"list<{t}>")
+
+ws_name = "narrative_1695311690413"
```

Comment:
If this is a real workspace, with real data used by VCR to set up its cassettes, we should probably put this in a config.